third_party/terraform/utils/retry_helpers.go

```diff
@@ -0,0 +1,16 @@
+package google
+
+import (
+	"strings"
+
+	"google.golang.org/api/googleapi"
+)
+
+func iamMemberMissing(err error) bool {
```

Comment:
nit: Do you mind commenting with a brief high level description of what case the fn checks for? While this one is easy enough to infer, starting the list of predicates/checks with a header comment will hopefully mean authors of future ones add them too 🙂